src/test/java/org/opentripplanner/apis/gtfs/PatternTestModel.java

```diff
@@ -0,0 +1,44 @@
+package org.opentripplanner.apis.gtfs;
+
+import static org.opentripplanner.transit.model._data.TransitModelForTest.id;
+
+import org.opentripplanner.transit.model._data.TransitModelForTest;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.network.Route;
+import org.opentripplanner.transit.model.network.StopPattern;
+import org.opentripplanner.transit.model.network.TripPattern;
+import org.opentripplanner.transit.model.site.RegularStop;
+import org.opentripplanner.transit.model.timetable.ScheduledTripTimes;
+import org.opentripplanner.transit.model.timetable.Trip;
+import org.opentripplanner.transit.service.StopModel;
+
+public class PatternTestModel {
```

Comment:
I think this class should be in a different package. Neither of the test classes you are using this for are in the gtfs api package.

Reply:
I've moved it next to `TransitModelForTest`. You could also argue that it should be part of that class but I'm not sure.